src/timeline/InputBar.cpp

```diff
@@ -937,13 +940,33 @@ InputBar::command(const QString &command, QString args)
                                 cache::getMembers(this->room->roomId().toStdString(), 0, -1));
     } else if (command == QLatin1String("converttoroom")) {
         utils::removeDirectFromRoom(this->room->roomId());
+    } else if (command == QLatin1String("ignore")) {
+        this->toggleIgnore(args, true);
+    } else if (command == QLatin1String("unignore")) {
+        this->toggleIgnore(args, false);
     } else {
         return false;
     }
 
     return true;
 }
 
+void
+InputBar::toggleIgnore(const QString &user, const bool ignored)
+{
+    UserProfile *profile = new UserProfile(QString(), user, TimelineViewManager::instance());
+    connect(profile, &UserProfile::failedToFetchProfile, [user] {
+        MainWindow::instance()->showNotification(tr("Failed to fetch user %1").arg(user));
```

Comment:
Please also delete the profile here :3

(Also you don't need to merge master always, it will get merged eventually anyway)

Reply:
Ah right, so used to smart pointers that I forget about branching when I can't use them :p

Reply:
(And I merged master to see if CI would be have better)